deep_quoridor/src/plugins/wandb_train.py

```diff
@@ -67,8 +69,29 @@ def __init__(
         self.agent_evolution_tournament = agent_evolution_tournament
         self.include_raw_metrics = include_raw_metrics
 
-    def _initialize(self, game):
-        assert self.agent
+        if wandb_run is None:
+            self.run = wandb.init(
```

Comment:
I moved run initialization into  the constructor because it was a bit unwieldy to wait until `start_game()` is called to start the run.